test/dotnet-restore.Tests/GivenThatIWantToRestoreApp.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+using FluentAssertions;
+using System.Linq;
+
+namespace Microsoft.DotNet.Restore.Tests
+{
+    public class GivenThatIWantToRestoreApp : TestBase
+    {
+        [Theory]
+        [InlineData("test1", "test1/", false)]
```

Comment:
I don't think we need 8 cases here, particularly since each costs ~10s. Can we do one positive test, and count on NuGet to test the specifics?